### PR TITLE
chore, fix(egen): corrections from template, guidelines, & fixes docker tag command

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
The docker tag command fails to run when the existing tag is not provided in Colab. The fix resolves it by considering the image name along with the tag from the arguments. 

Additionally, this PR includes the following updates:

1. Adds collapsed license comment.
2. Adds a link for Colab enterprise.
3. Replaces aip with aiplatform, Cloud ML with Vertex AI, & REGION with LOCATION (as per the template).
4. Reduces the future tense usage.
5. Heading style corrections.
6. Removes IS_TESTING from the cleaning up section.
7. Replaces K80 with T4 in the explanation about GPUs.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 1]
```diff
@@ -32,25 +34,27 @@
...
+        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Google Colaboratory logo\"><br> Open in Colab\n",
         "    </a>\n",
         "  </td>\n",
-        "  <td>\n",
-        "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/prediction/get_started_with_tf_serving.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
-        "      View on GitHub\n",
+        "  <td style=\"text-align: center\">\n",
+        "    <a href=\"https://console.cloud.google.com/vertex-ai/colab/import/https:%2F%2Fraw.githubusercontent.com%2FGoogleCloudPlatform%2Fvertex-ai-samples%2Fmain%2Fnotebooks%2Fofficial%2Fprediction%2Fget_started_with_tf_serving.ipynb\">\n",
+        "      <img width=\"32px\" src=\"https://lh3.googleusercontent.com/JmcxdQi-qOpctIvWKgPtrzZdJJK-J3sWE1RsfjZNwshCFgE_9fULcNpuXYTilIR2hjwN\" alt=\"Google Cloud Colab Enterprise logo\"><br> Open in Colab Enterprise\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:44]: update colab enterprise URL:

- https://cloud.google.com/ml-engine/images/colab-enterprise-logo-32px.png

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 2]
```diff
@@ -127,15 +131,22 @@
...
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "install_aip"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the following packages to execute this notebook."
+        "### Install Vertex AI SDK and other required packages\n"
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:149]: Vertex AI SDK for Python

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 3]
```diff
@@ -155,136 +166,92 @@
...
+        "id": "54c5ef8a8f43"
       },
-      "outputs": [],
       "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
-        "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:202]: remove "Please" and contract "it it"

- Wait until it's finished...

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 4]
```diff
@@ -388,16 +354,16 @@
...
-        "    (aip.gapic.AcceleratorType.NVIDIA_TESLA_K80, 4)\n",
+        "    (aiplatform.gapic.AcceleratorType.NVIDIA_TESLA_T4, 4)\n",
         "\n",
         "\n",
         "Otherwise specify `(None, None)` to use a container image to run on a CPU.\n",
         "\n",
         "Learn more about [hardware accelerator support for your region](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators).\n",
         "\n",
-        "*Note*: TF releases before 2.3 for GPU support will fail to load the custom model in this tutorial. It is a known issue and fixed in TF 2.3. This is caused by static graph ops that are generated in the serving function. If you encounter this issue on your own custom models, use a container image for TF 2.3 with GPU support."
+        "*Note*: TF releases before 2.3 for GPU support fail to load the custom model in this tutorial. It is a known issue and fixed in TF 2.3. This is caused by static graph ops that are generated in the serving function. If you encounter this issue on your own custom models, use a container image for TF 2.3 with GPU support."
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:366]: change to: "It's a known issue and fixed in TF 2.3" (i.e., involves contraction)

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 5]
```diff
@@ -683,7 +649,7 @@
       "source": [
         "## Upload the model for serving\n",
         "\n",
-        "Next, you will upload your TF.Keras model from the custom job to Vertex `Model` service, which will create a Vertex `Model` resource for your custom model. During upload, you need to define a serving function to convert data to the format your model expects. If you send encoded data to Vertex AI, your serving function ensures that the data is decoded on the model server before it is passed as input to your model.\n",
+        "Next, you upload your TF.Keras model from the custom job to Vertex `Model` service, which creates a Vertex `Model` resource for your custom model. During upload, you need to define a serving function to convert data to the format your model expects. If you send encoded data to Vertex AI, your serving function ensures that the data is decoded on the model server before it is passed as input to your model.\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:652]: contract "it is"

- before it's passed...

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 6]
```diff
@@ -700,7 +666,7 @@
         "\n",
         "Both the preprocessing and post-processing functions are converted to static graphs which are fused to the model. The output from the underlying model is passed to the post-processing function. The post-processing function passes the converted/packaged output back to the HTTP server. The HTTP server returns the output as the HTTP response content.\n",
         "\n",
-        "One consideration you need to consider when building serving functions for TF.Keras models is that they run as static graphs. That means, you cannot use TF graph operations that require a dynamic graph. If you do, you will get an error during the compile of the serving function which will indicate that you are using an EagerTensor which is not supported."
+        "One consideration you need to consider when building serving functions for TF.Keras models is that they run as static graphs. That means, you cannot use TF graph operations that require a dynamic graph. If you do, you get an error during the compile of the serving function which indicates that you are using an EagerTensor which is not supported."
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:669]: 1. change "cannot" to  "can't" (i.e., contraction)
2. change "you are" to "you're"
3.  change "is not" to "isn't"

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 7]
```diff
@@ -778,9 +744,9 @@
         "\n",
         "You can get the signatures of your model's input and output layers by reloading the model into memory, and querying it for the signatures corresponding to each layer.\n",
         "\n",
-        "For your purpose, you need the signature of the serving function. Why? Well, when we send our data for prediction as a HTTP request packet, the image data is base64 encoded, and our TF.Keras model takes numpy input. Your serving function will do the conversion from base64 to a numpy array.\n",
+        "For your purpose, you need the signature of the serving function. Why? Well, when we send our data for prediction as a HTTP request packet, the image data is base64 encoded, and our TF.Keras model takes numpy input. Your serving function does the conversion from base64 to a numpy array.\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:747]: shouldn't use "we"

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 8]
```diff
@@ -1214,7 +1178,7 @@
         "id": "batch_request_wait:mbsdk"
       },
       "source": [
-        "### Wait for completion of batch prediction job\n",
+        "## Wait for completion of batch prediction job\n",
         "\n",
         "Next, wait for the batch job to complete. Alternatively, one can set the parameter `sync` to `True` in the `batch_predict()` method to block until the batch prediction job is completed."
```
Comment (kittyabs) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:1183]: change from "one can" to "you can"

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 9]
```diff
@@ -902,9 +868,9 @@
         "id": "ca3fa3f6a894"
       },
       "source": [
-        "## Deploying `Model` resources to an `Endpoint` resource.\n",
+        "## Deploy `Model` resources to an `Endpoint` resource.\n",
         "\n",
-        "You can deploy one of more `Vertex AI Model` resource instances to the same endpoint. Each `Vertex AI Model` resource that is deployed will have its own deployment container for the serving binary. \n",
+        "You can deploy one of more `Vertex AI Model` resource instances to the same endpoint. Each `Vertex AI Model` resource that is deployed has its own deployment container for the serving binary. \n",
```
Comment (katiemn) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:873]: Should this be "one or more"?

File: notebooks/official/prediction/get_started_with_tf_serving.ipynb [hunk 10]
```diff
@@ -1032,7 +998,7 @@
         "\n",
         "Batch prediction provides the ability to do offline batch processing of large amounts of prediction requests. Resources are only provisioned during the batch process and then deprovisioned when the batch request is completed. The results are stored in Cloud Storage, in contrast to online prediction where the results are returned as a HTTP response packet.\n",
         "\n",
-        "The input format for your batch job is dependent on the format supported by your model server. Foremost, the web server in your model server must support a JSONL format, which the web server will convert to a format support either directly by the model input intertace or a serving function interface. For batch prediction, this JSONL format is referred to as the `pivot` format.\n",
+        "The input format for your batch job is dependent on the format supported by your model server. Foremost, the web server in your model server must support a JSONL format, which the web server converts to a format support either directly by the model input intertace or a serving function interface. For batch prediction, this JSONL format is referred to as the `pivot` format.\n",
```
Comment (katiemn) [inline notebooks/official/prediction/get_started_with_tf_serving.ipynb:1001]: Change intertace -> interface?

**Review by katiemn — APPROVED** [review]

**Pull request merged**